torchgeo/datasets/landcoverai.py

```diff
@@ -50,7 +52,224 @@ class LandCoverAI(NonGeoDataset):
 
     If you use this dataset in your research, please cite the following paper:
 
-    * https://arxiv.org/abs/2005.02264v3
+    * https://arxiv.org/abs/2005.02264v4
```

Comment:
Can you add:
```rst
.. versionadded:: 0.5
```
This will document that the base class didn't exist until torchgeo 0.5.0